src/components/Heading/Heading.module.css

```diff
@@ -1,10 +1,59 @@
 .container {
-  height: 8vh;
+  height: auto;
   width: 100vw;
-  background: #011b29;
+  background: transparent;
   display: flex;
-  justify-content: center;
+  justify-content: space-around;
   align-items: center;
   font-weight: 700;
```

Comment:
Let's add some padding to the sides and reduce the font size on smaller screens.